content/help/ops/security/repairing-citadel/index.md

```diff
@@ -5,6 +5,15 @@ weight: 10
 keywords: [security,citadel,ops]
 ---
 
+`Currently we only support single Citadel instance. Running multiple Citadel instances may introduce
```

Comment:
Backticks is for code sequences. Instesd, you should wrap the paragraph as an info block:

{{< info >}}
Text goes here
{{< /info >}}

Reply:
only support a single Citadel

Reply:
done.